sys/netgraph/ng_pipe.c

```diff
@@ -976,7 +998,7 @@ ngp_modevent(module_t mod, int type, void *unused)
 		    sizeof (struct ngp_fifo)), NULL, NULL, NULL, NULL,
 		    UMA_ALIGN_PTR, 0);
 		if (ngp_zone == NULL)
-			panic("ng_pipe: couldn't allocate descriptor zone");
+			error = ENOMEM;
```

Comment:
This check can simply be removed: uma_zcreate() never fails.

Reply:
A quick grep shows that there are various places where the result of `uma_zcreate()` is checked (e.g. [here](https://github.com/freebsd/freebsd-src/blob/464b1ab216d5f78054626524addf99f5c622da96/sys/dev/dpaa/if_dtsec_rm.c#L313-L316) or [here](https://github.com/freebsd/freebsd-src/blob/464b1ab216d5f78054626524addf99f5c622da96/sys/dev/wg/if_wg.c#L2994-L2996)).  How are they different?

Reply:
They aren't different, that code doesn't need to check either. It's just cargo-culting.

Reply:
Removed